src/argilla/server/schemas/v1/questions.py

```diff
@@ -92,7 +92,13 @@ class Config:
 
 class TextQuestionSettingsUpdate(UpdateSchema):
     type: Literal[QuestionType.text]
-    use_markdown: bool
+    use_markdown: Optional[bool]
```

Comment:
Shouldn't this just be set to False?

```suggestion
    use_markdown: Optional[bool] = False
```

Reply:
Ok it cannot be False because it's to update the existing, thanks for the internal clarification!